introducing-pgstream.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: 'Introducing pgstream: Postgres replication with DDL changes'
+description: 'Today we’re excited to expand our open source Postgres platform with pgstream, a CDC command line tool and library for PostgreSQL with replication support for DDL changes to any provided output.'
+image:
+  src: /images/pgstream/introducing-pgstream@2x.jpg
```

Comment:
@miukimiu @snide I'm guessing this will work, but we typically use the GitHub link or a Xata link. Any issues with this?

Reply:
I followed @miukimiu's advice on this one:
https://xata-hq.slack.com/archives/C0530CRV3FT/p1720778008386189?thread_ts=1720775805.192319&cid=C0530CRV3FT

Reply:
<img width="364" alt="Screenshot 2024-07-17 at 10 00 30" src="https://github.com/user-attachments/assets/b02c1091-e93f-4284-9f68-ffb5fb0fde24">

</br>
</br>
Hey @eminano, I was referring to the images outside of the frontmatter. The images that were inside the MDX image tags. The build was falling because those image tags were causing the errors. So to fix it I suggested using relative src paths.

For the `image` and `ogImage` we normally get the src from GitHub like `https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgstream/introducing-pgstream@2x.jpg`.

Reply:
Ah sorry, I misunderstood! I will update them accordingly 🙏